articlequality/extractors/ptwiki.py

```diff
@@ -10,12 +10,12 @@ def from_template(template):
     project_name = normalize_project_name(template.name)
     if project_name == "marca de projeto":
```

Comment:
It seems weird to treat "marca de projeto" (which is the template name) as if it was the name of a some wikiproject (which is implied by the current variable name).